src/spikeinterface/extractors/nwbextractors.py

```diff
@@ -174,14 +170,20 @@ def read_nwbfile(
     if file_path is None and file is None:
         raise ValueError("Provide either file_path or file")
 
-    hdf5_file = _read_hdf5_file(
-        file_path=file_path,
-        file=file,
-        stream_mode=stream_mode,
-        cache=cache,
-        stream_cache_path=stream_cache_path,
-    )
-    io = NWBHDF5IO(file=hdf5_file, mode="r", load_namespaces=True)
+    if file_path is not None and stream_mode is None:
```

Comment:
But for your specific file this general fix will determine the behavior as `_read_hdf5_file` will not be called.